### PR TITLE
#169589914 : Set up Winston logger

### PR DESCRIPTION
**Pull request by Mastel22**
#### What does this PR do?
Sets up the winston logger for the project
#### Description of Task to be completed?

- [X] install winston package
- [X] create own logger

#### How should this be manually tested?

- Git clone this repo
- run `npm install`
- run `npm run winston`

#### Any background context you want to provide?
-N/A
#### What are the relevant pivotal tracker stories?
[#169589914](https://www.pivotaltracker.com/story/show/169589914)
#### Questions:
None

### CODE REVIEW

File: src/utils/winston.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const { combine, timestamp, prettyPrint } = format;
+const logger = createLogger({
+  format: combine(
+    timestamp(),
+    prettyPrint(),
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: './error.log', level: 'error' }),
```
Comment (timolinn) [inline src/utils/winston.js]: Really good job here Mastel. We need a `logs` folder at the project root which would contain both files. The log files should be `.gitignore`d, once that is done we need a way to initialise the log files automatically. I suggest we write a short script to create these log files when they don't exist yet.

Comment (Mastel22) [reply src/utils/winston.js]: Okay. Let me work on that now

File: src/utils/winston.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const { combine, timestamp, prettyPrint } = format;
+const logger = createLogger({
+  format: combine(
+    timestamp(),
+    prettyPrint(),
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: '.logs/error.log', level: 'error' }),
+    new transports.File({ filename: './logs/info.log', level: 'info' }),
```
Comment (b0nbon1) [inline src/utils/winston.js]: Great job but I have tested it but the error log creates the  `.logs/error.log` which works okay but the info does not so I suggest maybe the line `./logs/info.log` to `.logs/info.log` for it to work.

**Review by b0nbon1 — CHANGES_REQUESTED** [review]

File: src/utils/winston.js (hunk 2 above)
Comment (b0nbon1) [reply src/utils/winston.js]: And also on the pr description how to manually  you should add the script to run so as to test it

Comment (Mastel22) [reply src/utils/winston.js]: Oh yeah! Missed that. Fixed now. Thanks

**Review by b0nbon1 — DISMISSED** [review]
Great job, keep it up.

**Review by nshaphan — DISMISSED** [review]
great work!

**Review by b0nbon1 — DISMISSED** [review]

**Review by nshaphan — DISMISSED** [review]

**Review by b0nbon1 — DISMISSED** [review]

**Review by nshaphan — DISMISSED** [review]

**Review by timolinn — APPROVED** [review]

**Review by b0nbon1 — APPROVED** [review]

**Review by nshaphan — APPROVED** [review]

**Pull request merged**